test/suites/backup.sh

```diff
@@ -103,8 +103,10 @@ EOF
     echo "hello world" | lxc exec c1 --project test -- tee /mnt/test.txt
     [ "$(lxc exec c1 --project test -- cat /mnt/test.txt)" = "hello world" ]
     lxc stop -f c1
+    lxc config set c1 snapshots.expiry 1d
     lxc snapshot c1
     lxc info c1
+    snapshotExpiryDateBefore=$(lxc info c1 | grep "snap0" | awk -F '|' '{print $4' | xargs)
```

Comment:
This has a bogus awk script but how about simply saving the whole `snap0` info for comparison?

```
snapshotExpiryDateBefore="$(lxc info c1 | grep -wF "snap0")"
```

Reply:
Thanks for pointing that out!